python/tvm/relay/frontend/tensorflow.py

```diff
@@ -2027,6 +2081,8 @@ def _impl(inputs, attr, params, mod):
     'Mod'                               : _elemwise('mod'),
     'Mul'                               : _elemwise('multiply'),
     'Neg'                               : AttrCvt('negative'),
+    'NonMaxSuppressionV2'               : _nms(),
+    'NonMaxSuppressionV3'               : _nms(),
```

Comment:
What's the difference between V2 and V3? Do we need to treat them differently?

Reply:
V3 adds an extra argument: score_threshold. This has already been handled.